apiary_v2.apib

```diff
@@ -981,15 +1057,24 @@ After an order has been created, this endpoint can be used to retrieve order dat
 
     + Body
 
-          {}
-
-+ Response 400 (application/json)
-    
-    + Body
-    
           {
-             "code": "bad_request",
-             "message": "Request payload is not valid.",
+            "id": "2c9c3662-d679-4e13-bfdb-7c2580c80b46",
+            "reference": "UMD12345",
+            "created": "2024-07-28T18:05:36.806548Z",
+            "submitted_at": "2024-07-28T18:05:53.714553Z",
+            "num_items": 12,
+            "order_items": "https://partner.embed.unmade.com/v2/orders/2c9c3662-d679-4e13-bfdb-7c2580c80b46/items/",
+            "shipping_address": "https://partner.embed.unmade.com/v2/orders/2c9c3662-d679-4e13-bfdb-7c2580c80b46/shipping_address/",
+            "manufacturing_order": "https://factory.unmade.com/v2/orders/fd792895-1e32-41b1-90af-683f0cf9f753/",
+            "partner_data": {
+              "my_partner_data_key": "foo"
+            },
+            "state": "submitted",
+            "tracking_number": "",
+            "is_order_portal": false,
+            "is_test_order": false,
+            "ready_to_ship_at": null,
+            "shipped_at": null
```

Comment:
Nice!! this reminds me could be interesting an endpoint `status` or `state` for the ecommerce integrations. A quick and simple endpoint to respond, this can be added later just sharing the idea.